addons/web/static/src/views/graph/graph_model.js

```diff
@@ -506,7 +514,7 @@ export class GraphModel extends Model {
                     if (type === "boolean") {
                         label = `${val}`; // toUpperCase?
                     } else if (val === false) {
-                        label = this.env._t("Undefined");
+                        label = this._getDefaultFilterLabel(groupBy);
```

Comment:
Why not passing the fieldName or just the groupBy instead of the whole fields in the groupby?

```suggestion
                        label = this._getDefaultFilterLabel(gb);
```

or 

```suggestion
                        label = this._getDefaultFilterLabel(gb.fieldName);
```

Reply:
Changes Done, Thanks :smiley: